easybuild/easyconfigs/g/GraPhlAn/GraPhlAn-1.1.3-foss-2019b-Python-2.7.16.eb

```
@@ -0,0 +1,42 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage: 	https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::	Robert Qiao <rob.qiao@flinders.edu.au>
+# License::	MIT
+#
+# Notes::	requires python 2.7, biopython 1.6 or higher and matplotlib 1.1 or higher
+##
+
+easyblock = 'Tarball'
+
+name = 'GraPhlAn'
+version = '1.1.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://segatalab.github.io/tools/graphlan'
+description = """GraPhlAn is a software tool for producing high-quality circular representations
+ of taxonomic and phylogenetic trees. It focuses on concise, integrative, informative,
+ and publication-ready representations of phylogenetically- and taxonomically-driven investigation.
+"""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+source_urls = ['https://github.com/biobakery/%(namelower)s/archive']
+sources = ['%(version)s.tar.gz']
+checksums = ['dddca54ec308506924cc46482338a9ab93f430b4d1ea479bd8bc30eb91edaf22']
+
+dependencies = [
+    ('Python', '2.7.16'),
+    ('Biopython', '1.75', '%(versionsuffix)s'),
+    ('matplotlib', '2.2.4', '%(versionsuffix)s'),
```

this can be simplified a bit:
```suggestion
    ('Biopython', '1.75', versionsuffix),
    ('matplotlib', '2.2.4', versionsuffix),
```